batch/sc-batch-fargate.yaml

```diff
@@ -57,30 +57,30 @@ Parameters:
       - true
       - false
   Command:
-    Type: List<String>
+    Type: String
     Description: >
-      The list of commands to pass to the container image (i.e. ["echo", "hello", "world"])
+      A pipe seperated list of commands to pass to the container image (i.e. echo|hello|world)
```

Comment:
typo: sep`a`rated
... and it's not a list of commands.  In the example 'world' isn't a command.  Instead you could say something like, 'A pipe separated list which is the command to pass to the container.'

Reply:
done